package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,813 @@
+        start : int (optional)
+            starting frame-index for analysis, ``None`` is the first one, 0.
+            `start` and `stop` are 0-based frame indices and are used to slice
+            the trajectory (if supported) [``None``]
+        stop : int (optional)
+            last trajectory frame for analysis, ``None`` is the last one [``None``]
+        step : int (optional)
+            read every `step` between `start` (included) and `stop` (excluded),
+            ``None`` selects 1. [``None``]
+        detect_hydrogens : {"distance", "heuristic"} (optional)
+            Determine the algorithm to find hydrogens connected to donor
+            atoms. Can be "distance" (default; finds all hydrogens in the
+            donor's residue within a cutoff of the donor) or "heuristic"
+            (looks for the next few atoms in the atom list). "distance" should
+            always give the correct answer but "heuristic" is faster,
+            especially when the donor list is updated each
+            for each frame. ["distance"]
+        distance_type : {"hydrogen", "heavy"} (optional)
+            Measure hydrogen bond lengths between donor and acceptor heavy
+            attoms ("heavy") or between donor hydrogen and acceptor heavy
+            atom ("hydrogen"). If using "heavy" then one should set the *distance*
+            cutoff to a higher value such as 3.5 Å. ["hydrogen"]
+        debug : bool (optional)
+            If set to ``True`` enables per-frame debug logging. This is disabled
+            by default because it generates a very large amount of output in
+            the log file. (Note that a logger must have been started to see
+            the output, e.g. using :func:`MDAnalysis.start_logging`.)
+        verbose : bool (optional)
+            Toggle progress output. (Can also be given as keyword argument to
+            :meth:`run`.)
+
+        Raises
+        ------
+        :exc:`SelectionError`
+            is raised for each static selection without the required
+            donors and/or acceptors.
+
+        Notes
+        -----
+        In order to speed up processing, atoms are filtered by a coarse
+        distance criterion before a detailed hydrogen bonding analysis is
+        performed (`filter_first` = ``True``). This is now the default.
+
+        If your selections will essentially remain the same for all time steps
+        (i.e. residues are not moving farther than 3 x `distance`), for
+        instance, if no large conformational changes are involved then you can improve
+        performance by setting the `update_selection1` and/or `update_selection2`
+        keywords to ``False``.
+
+        """
+        self.water_selection = water_selection
+        self.update_water_selection = update_water_selection
+        HydrogenBondAnalysis.__init__(self,universe, selection1, selection2, selection1_type, update_selection1,
```

Comment:
1. pep8: space after comma
2. when you are calling another class with keyword arguments *never* rely on the order of these kwargs. In order to write robust code you *must* call kwargs as named kwargs, not by order. This is very fragile. kwargs never promise that the order remains the same. Please fix.
3. remove `detect_hydrogens`: use the `HydrogenBondAnalysis` default (see #1668)

Reply:
@orbeckst Thank you for the suggestion. kwarg removed.